src/App.tsx

```diff
@@ -2,98 +2,13 @@
 import React from 'react';
 
 import './App.scss';
```

Comment:
remove unnecessary line
```suggestion
import './App.scss';
```